ui/component/sideNavigation/view.jsx

```diff
@@ -278,20 +312,38 @@ function SideNavigation(props: Props) {
             })}
           </ul>
 
-          {sidebarOpen && isPersonalized && subscriptions && subscriptions.length > 0 && (
-            <ul className="navigation__secondary navigation-links navigation-links--small">
-              {subscriptions.map(({ uri, channelName }, index) => (
-                <li key={uri} className="navigation-link__wrapper">
-                  <Button
-                    navigate={uri}
-                    label={channelName}
-                    className="navigation-link"
-                    activeClass="navigation-link--active"
-                  />
-                </li>
-              ))}
-            </ul>
-          )}
+          {sideList === SIDE_LIST.FOLLOWED_TAGS &&
+            sidebarOpen &&
+            isPersonalized &&
+            followedTags &&
+            followedTags.length > 0 && (
+              <ul className="navigation__secondary--tags tags--vertical">
+                {followedTags.map(({ name }, key) => (
+                  <li className="navigation-link__wrapper" key={name}>
+                    <Tag navigate={`/$/tags?t${name}`} name={name} />
```

Comment:
Instead of `<Tag />``, can you make this a button with the same style as channels, and add a  `#` in front of the tag name, like hashtags.

You will have to remove the `activeClass` prop since they all go to the same page.